lib/azure/armrest/configuration.rb

```diff
@@ -266,22 +266,7 @@ def set_provider_api_versions
       end
 
       def fetch_providers
-        uri = Addressable::URI.join(Azure::Armrest::RESOURCE, 'providers')
-        uri.query = "api-version=#{api_version}"
-
-        response = ArmrestService.send(
-          :rest_get,
-          :url         => uri.to_s,
-          :proxy       => proxy,
-          :ssl_version => ssl_version,
-          :ssl_verify  => ssl_verify,
-          :headers     => {
-            :content_type  => content_type,
-            :authorization => token
-          }
```

Comment:
@bronaghs Yes. The :ssl_verify accessor doesn't have an explicit default, so it defaults to nil. This is deliberate so that it defaults to whatever RestClient uses.